Cargo.toml

```diff
@@ -1,19 +1,20 @@
 [package]
 name = "yaiwr"
-version = "0.0.1"
```

Comment:
I increased the version for my own sake - tracking changes in this repo
What do you mean by smantic version here? Version is in semver format: 0 = major, 0 = minor, 2=patch

Reply:
Yep, we shouldn't increment the version for each PR, because that causes problems for repositories with multiple people working on them. We should only bump when we make a release.

Reply:
ok

Reply:
version restored to 0.0.1 https://github.com/softdevteam/pavel.yaiwr/pull/7/commits/5b40cf153e00d0c0cf4e7ccf6caaee33a45e2c4c